src/read-models/recipient-read-model.ts

```diff
@@ -49,4 +49,25 @@ export class RecipientReadModel {
       entity.referralSheetUrl
     )
   }
+
+  @Projects(Recipient, 'identityDocumentNumber')
+  public static projectRecipientUserByDocumentNumber(entity: Recipient): ProjectionResult<RecipientReadModel> {
```

Comment:
why do you call this `entity`?

Reply:
It's how boost cli names that param by default!